app/models/match.rb

```diff
@@ -54,6 +54,16 @@ def status
     status_updates.last.status
   end
 
+  def status_class
+    return "" unless status_updates.last
+    status_updates.last.good? ? "progress-bar-success" : "progress-bar-danger"
+  end
+
+  def progress
+    return 0 unless concluded_at.present? && started_at.present?
+    100 * ((Time.zone.now - started_at) / (concluded_at - started_at))
```

Comment:
That's kinda neat, that you can subtract a time object from another time object... Guess it converts it to a timestamp or something?